services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -72,6 +74,10 @@ class ComputationalBackendSettings(BaseCustomSettings):
     COMPUTATIONAL_BACKEND_ENABLED: bool = Field(
         default=True,
     )
+    COMPUTATIONAL_BACKEND_SCHEDULING_CONCURRENCY: PositiveInt = Field(
```

Comment:
maybe ask expose this in the `.env-devel` and relative OPS branches so we can tune it easily on production

Reply:
precisely I do not want to expose it at the moment. I want to play in master if needed.